docs/internals/contributing/triaging-tickets.txt

```diff
@@ -420,3 +420,55 @@ the ticket database:
 .. _`easy pickings`: https://code.djangoproject.com/query?status=!closed&easy=1
 .. _`creating an account on Trac`: https://www.djangoproject.com/accounts/register/
 .. _password reset page: https://www.djangoproject.com/accounts/password/reset/
+
+Bisecting a regression
+----------------------
+
+If you are reporting a bug or attempting to reproduce a reported bug in Django,
+an extremely helpful piece of information is the commit that introduced the
+regression. Knowing the commit that caused the change in behavior helps
+identify if the change was intentional or if it was an inadvertent side-effect.
+Here's how you can help out with this.
+
+Begin by writing a regression test for Django's test suite for the issue. For
+example, we'll pretend we're debugging a regression in migrations. After you've
+written the test and confirmed that it fails on the latest master, put it in a
+separate file that you can run standalone. For our example, we'll pretend we
+created ``tests/migrations/test_regression.py``, which can be run with:
+
+.. code-block:: bash
+
+    $ ./runtests.py migrations.test_regression
+
+Next, we mark the current point in history as being "bad" since the test fails:
+
+.. code-block:: bash
+
+    $ git bisect bad
+    You need to start by "git bisect start"
+    Do you want me to do it for you [Y/n]? y
+
+Now, we need to find a point in git history before the regression was
+introduced. Use something like ``git co HEAD~100`` to checkout an earlier
+revision (100 commits earlier, in this case). Check if your test fails. If so,
+checkout an earlier revision and recheck. Once you find a revision where your
```

Comment:
Shouldn't this commit be marked as bad instead? Saves 100 commits from being bisected.